### DISCUSSION TITLE
Crash with upgrades

### DISCUSSION THREAD

**Discussion by OldAntiDotes** [P3pp3rF1y/SophisticatedBackpacks#1111] [Q&A] [answered]
It crashes when I try to insert some improvements, I can give more details if necessary, but write what exactly. I will also attach crash reports
[crash-2024-08-10_13.41.38-client.txt](https://github.com/user-attachments/files/16569439/crash-2024-08-10_13.41.38-client.txt)

**Comment by P3pp3rF1y** [comment, marked as answer]
You have newest core but old version of backpacks just update backpacks to latest and issue is solved

**Reply by OldAntiDotes** [reply]
Oh, I'll try, thanks.

**Reply by OldAntiDotes** [reply]
Yes, it helped, thank you very much again
